src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.kt

```diff
@@ -215,29 +226,30 @@ class SonarLintRulePanel(private val project: Project) : JBLoadingPanel(BorderLa
         sectionsTabs.insertTab(tabDesc.title, null, sectionPanel, null, index)
     }
 
-    private fun updateHeader(ruleDescription: ActiveRuleDetailsDto) {
+    private fun updateHeader(finding: Finding, ruleDescription: ActiveRuleDetailsDto) {
         ruleNameLabel.text = ruleDescription.name
-        headerPanel.update(ruleDescription.key, ruleDescription.type, ruleDescription.severity)
-    }
+        securityHotspotHeaderMessage.isVisible = finding is LiveSecurityHotspot
+        if (finding is LiveSecurityHotspot) {
+            val htmlStringBuilder = StringBuilder("""
+                A <a href="https://docs.sonarqube.org/latest/user-guide/security-hotspots/">Security Hotspot</a> is not necessarily an issue, but a security-sensitive piece of code that needs a manual review.
+                <br>
+                At the moment, the status of a Security Hotspot can only be updated in SonarQube. 
+                """.trimIndent())
+            if (finding.serverFindingKey != null) {
+                val serverConnection =
+                    SonarLintUtils.getService(project, ProjectBindingManager::class.java).serverConnection
+                val projectKey = Settings.getSettingsFor(project).projectKey
+
+                htmlStringBuilder.append("""
+                Click <a href="${serverConnection.hostUrl}/security_hotspots?id=${projectKey}&hotspots=${finding.serverFindingKey}">here</a>
```

Comment:
Parameters are not encoded with UTF-8, not sure if that's really an issue nowadays

Reply:
You mean escaped for URL compliance. You are right!